README.md

```diff
@@ -54,6 +54,9 @@ The following variants support EKS, as described above:
 - `aws-k8s-1.19`
 - `aws-k8s-1.20`
 - `aws-k8s-1.21`
+- `aws-k8s-1.21-nvidia`
+
+Please refer to [this document](QUICKSTART-EKS.md#aws-k8s-121-nvidia-variant) to learn more about the `aws-k8s-1.21-nvidia` variant.
```

Comment:
I'd prefer not to link to variant specifics here, or else we'll be doing that everywhere on this page. In general, we need to figure out a more structured approach to getting started with particular variants, since this isn't scaling very well.